extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss

```diff
@@ -15,17 +16,18 @@ $close-button-background: '/resources/wikia/ui_components/modal/images/close-dar
 	background: url('/extensions/wikia/PotentialMemberPageExperiments/images/858-px-background.png') no-repeat center bottom;
 	background-size: cover;
 	border-bottom: 2px solid $color-blue;
-	height: 150px;
 	margin: 18px 0;
 	overflow: hidden;
+	padding-bottom: $banner-bottom-space;
```

Comment:
I added this for glue banner, so should be same size as previously `height: calc(320px - #{$banner-bottom-space});`